verifier/src/tcb.rs

```diff
@@ -197,14 +219,13 @@ pub struct TcbComponent {
 /// Due to the way the TCB info is signed the contents should be provided as is.
 #[derive(Debug, Deserialize, Clone)]
 #[serde(rename_all = "camelCase")]
-pub struct TcbInfoRaw<'a> {
-    #[serde(borrow)]
-    tcb_info: &'a RawValue,
+pub struct TcbInfoRaw {
```

Comment:
Thoughts on changing `TcbInfoRaw` to  `SignedTcbInfo` for symmetry with `SignedQeIdentity`.
It's mainly a cosmetic change, which does touch the public API, but it may not be that beneficial.

Reply:
I personally like `SignedTcbInfo` and I don't think a breaking public API change is a problem right now

Reply:
Sounds good, I'll create a follow on PR that does the renaming after this and it's parent are merged